pep.rst

```diff
@@ -240,6 +240,21 @@ Therefore these prefixes can't be used as a tag:
 
     bytesprefix: "b" | "B" | "br" | "Br" | "bR" | "BR" | "rb" | "rB" | "Rb" | "RB"
 
+A tag name can also be any *dotted* name, for example ``app.html``. By extension, this also
+means ``app.f`` would be allowed:
```

Comment:
While it's okay for this to be allowed, we should probably discourage users from doing this. It feels too easy to read this as an f-string to me.

```suggestion
means ``app.f`` would be allowed, but discouraged:
```

Reply:
You're right, that choice was made intentionally to show the weird consequence. But we shouldn't show it. I will change to `app.html`.